src/core/netplay.cpp

```diff
@@ -48,13 +48,25 @@ enum class ControlMessage : u32
   SynchronizeComplete,
 };
 
+enum class SessionMessage : u32
+{
+  ChatMessage,
```

Comment:
Any reason why this couldn't be a control message? (would save allocating another channel)

Reply:
Was simply an assumption. If it being a control message wont hinder future plans than it isnt a problem. it can indeed just be a control message then. will fix when i wakeup tomorrow 👍

Reply:
Perhaps my naming wasn't the best, I basically split into "ggpo" and "non-ggpo" (control) packet types. The latter doesn't require the ggpo session to be valid, so it can be used while resynchronizing.